apps/frontend/src/structure/ErrorBoundary.tsx

```diff
@@ -12,7 +12,7 @@ type ErrorBoundaryProps = { children: ReactNode } & Partial<
 
 const ErrorBoundary: React.FC<ErrorBoundaryProps> = ({
   children,
-  sentryReporting = false,
+  sentryReporting = true,
```

Comment:
Maybe this should be noSentryReporting so we can just add noSentryReporting to error boundaries